scripts/build_info.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+HEAD=$(git rev-parse --verify HEAD --short)
+DATE=$(date +%Y-%m-%d)
+TIME=$(date +%H:%M:%S)
+COMMITS=$(git log --no-merges --pretty=format:'%h|%s|%aN' -n 10)
```

Comment:
Showing the last 10 is rather arbitrary. What about showing the commits since the last release?